scene/resources/atlas_texture.cpp

```diff
@@ -227,8 +228,8 @@ bool AtlasTexture::is_pixel_opaque(int p_x, int p_y) const {
 		return true;
 	}
 
-	int x = p_x + region.position.x - margin.position.x;
-	int y = p_y + region.position.y - margin.position.y;
+	int x = p_x + rounded_region.position.x - margin.position.x;
+	int y = p_y + rounded_region.position.y - margin.position.y;
```

Comment:
Not necessary as positions are the same.
Same above.

Reply:
Yeah I used it for consistency.